### DISCUSSION TITLE
Dialog is empty after second click

### DISCUSSION THREAD

**Discussion by 8p-design** [Extended-Thunder/send-later#524] [Q&A] [answered]
If I click the TB Send button, the Send Later dialog opens fine.
If I click out to dismiss the dialog, and click the TB Send button again, the dialog is **empty**, no button, nothing.

I can still click where the button is "supposed" to be, to send out now, but the dialog is empty.

https://drive.google.com/open?id=1w-EeKi2MhMHgcapMFTqEJtATkKlZbebg&authuser=thierry.lavallee%40gmail.com&usp=drive_fs

**Comment by jikamens** [comment]
Are you still seeing this issue? If so, what platform are you on?

**Comment by jikamens** [comment]
Also, if you're still seeing this issue, what Thunderbird and Send Later version do you have?

**Comment by Ochope** [comment, marked as answer]
The problem seems to be gone in 9.2.9 ! :) 
Thanks for your work Jonathan.
